### ISSUE TITLE
Compile 5.05 on Centos6.4 x64 failed

### ISSUE THREAD

**Issue by ximply** [happyfish100/fastdfs#10]
I have install libfastcommon, but failed to compile fastdfs
and out put as follow:
/root/fastdfs-5.05/tracker/fdfs_trackerd.c:104: undefined reference to `g_current_time'
/root/fastdfs-5.05/tracker/fdfs_trackerd.c:105: undefined reference to `g_current_time'
/root/fastdfs-5.05/tracker/fdfs_trackerd.c:108: undefined reference to `log_init2'
/root/fastdfs-5.05/tracker/fdfs_trackerd.c:111: undefined reference to `get_base_path_from_conf_file'
/root/fastdfs-5.05/tracker/fdfs_trackerd.c:120: undefined reference to `process_action'
/root/fastdfs-5.05/tracker/fdfs_trackerd.c:194: undefined reference to `write_to_pid_file'
/root/fastdfs-5.05/tracker/fdfs_trackerd.c:347: undefined reference to `log_notify_rotate'
/root/fastdfs-5.05/tracker/fdfs_trackerd.c:355: undefined reference to `log_set_keep_days'
/root/fastdfs-5.05/tracker/fdfs_trackerd.c:360: undefined reference to `log_delete_old_files'
/root/fastdfs-5.05/tracker/fdfs_trackerd.c:437: undefined reference to `delete_pid_file'
......
.......

**Comment by huishi0823** [comment]
Have you resolved this issue?

**Comment by ximply** [comment]
No, finally, I gave up. Could you offer me some advice? Thank you in advance.

**Comment by huishi0823** [comment]
Maybe you should first compile & install libfastcommon-master.zip, then
ln -s /usr/lib64/libfastcommon.so /usr/local/lib/libfastcommon.so
ln -s /usr/lib64/libfastcommon.so /usr/lib/libfastcommon.so
ln -s /usr/lib64/libfdfsclient.so /usr/local/lib/libfdfsclient.so
ln -s /usr/lib64/libfdfsclient.so /usr/lib/libfdfsclient.so
( if /usr/local/lib/libfastcommon.so exists, rm -f /usr/local/lib/libfastcommon.so ...first)

**Comment by todsy95** [comment]
@huishi0823 It doesn't work.

**Comment by todsy95** [comment]
U can try this command: 
```
bash
yum install -y glibc.i686
```
